recipe/meta.yaml

```diff
@@ -48,8 +48,6 @@ outputs:
         - {{ pin_subpackage("arrow-cpp", max_pin="x.x.x") }}
       ignore_run_exports:
         - cudatoolkit
```

Comment:
```suggestion
      ignore_run_exports_from:
        - {{ compiler('cuda') }}
        - boost-cpp >=1.70
```

Reply:
`boost-cpp` doesn't have a `run_exports` so this section should not be needed at all.